plugin/adsk/plugin/adskMaterialCommands.cpp

```diff
@@ -16,7 +16,9 @@
 #include "adskMaterialCommands.h"
 
 #include <mayaUsd/ufe/Global.h>
+#include <mayaUsd/ufe/UsdSceneItem.h>
```

Comment:
Please add documentation for the material commands [here](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/commands/Readme.md).